### PR TITLE
Align Website User guide with Readme docs

### PR DESCRIPTION
**Pull request by Ayelet41**
Aligned the content from the old user guide (docs/Readme.md), the Tutorial and the new User Guide( website).
- This included changes that were made between the versions like "SLIs, SLOs and Burn rate Alerts" documentation that was added after the creation of the website.
- Updated links that refer to the old docs/readme file.
- Change the name of the "Learn Ops management"  to "Learn Cloud Operations" to reflect previous changes. 
- Remove SLO Monitoring step by steps images from Neo and update the path to be accessed from cloud shell

### CODE REVIEW

**Comment by Ayelet41** [comment]
Replacement for #697

**Comment by Crash-GHaun** [comment]
Lgtm! Will need to generate the website once submitted

File: website/content/en/docs/User Guide/Learn Cloud Operations/cloud_slo_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+2.Then you will set your SLI, you need to choose SLI type and the method(request vs window based):
+![image](/docs/images/user-guide/42-checkoutservice-sli.png)
+3. Then you will define your metric and you can also preview its performance based on historical data:
+![image](/docs/images/user-guide/41-checkoutservice-define-sli.png)
+4. Then you will configure your SLO, your target in a specific time window. You can also choose between [rolling window or a calendar window](https://sre.google/workbook/implementing-slos/):
+![image](/docs/images/user-guide/43-set-slo.png)
+
+#### Configure Burn Rate Alerts
+
+After you create the SLO, you can create[Burn Rate Alerts](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/alerting-on-budget-burn-rate)for those.
```
Comment (simonz130) [inline website/content/en/docs/User Guide/Learn Cloud Operations/cloud_slo_monitoring.md:84]: nit: a space between `you can create` and `[Burn Rate Alerts]`

**Review by simonz130 — APPROVED** [review]
This is FANTASTIC! Thank you Ayelet!

File: docs/tutorial.md [hunk 2]
```diff
@@ -197,48 +197,33 @@ You can find all the services under [monitoring → services → Services Overvi
...
-![image](./images/user-guide/39-checkout-service.png )
-2.Then we will set our SLI, we need to choose SLI type and the method(request vs window based):
-![image](./images/user-guide/42-checkoutservice-sli.png)
-3. Then we wil define our metric and we can also preview it's performance based on historical data:
-![image](./images/user-guide/41-checkoutservice-define-sli.png)
-4. Then we will configure our SLO, our target in a specific time window. We can also choose between [rolling window or a calendar window](https://sre.google/workbook/implementing-slos/):
-![image](./images/user-guide/43-set-slo.png)
+1. In the service screen we will choose Create SLO
+2.Then we will set our SLI, we need to choose SLI type and the method(request vs window based)
+3. Then we wil define our metric and we can also preview it's performance based on historical data
```
[the remark this answers is not in the document]
Comment (Ayelet41) [reply docs/tutorial.md:213]: Fixed

File: docs/tutorial.md [hunk 3]
```diff
@@ -197,48 +197,33 @@ You can find all the services under [monitoring → services → Services Overvi
...
-1. In the service screen we will choose Create SLO:
-![image](./images/user-guide/39-checkout-service.png )
-2.Then we will set our SLI, we need to choose SLI type and the method(request vs window based):
-![image](./images/user-guide/42-checkoutservice-sli.png)
-3. Then we wil define our metric and we can also preview it's performance based on historical data:
-![image](./images/user-guide/41-checkoutservice-define-sli.png)
-4. Then we will configure our SLO, our target in a specific time window. We can also choose between [rolling window or a calendar window](https://sre.google/workbook/implementing-slos/):
-![image](./images/user-guide/43-set-slo.png)
+1. In the service screen we will choose Create SLO
+2.Then we will set our SLI, we need to choose SLI type and the method(request vs window based)
```
[the remark this answers is not in the document]
Comment (Ayelet41) [reply docs/tutorial.md:212]: Fixed

**Pull request merged**